app/view/twig/_base/_listing.twig

```diff
@@ -301,13 +301,13 @@ row_header:     prop.extended and (prop.first or (prop.has_groupname and prop.ne
                     {% for taxonomyslug, values in content.taxonomy %}
                         {% if values|length > 1 %}
                             <li>
-                                <a class="nolink">{{ config.get('taxonomy')[taxonomyslug].name }}:
+                                <a class="nolink">{{ app.config.get('taxonomy')[taxonomyslug].name }}:
```

Comment:
That was my reading/remembering of https://github.com/bolt/bolt/blob/release/3.3/src/Twig/Extension/BoltExtension.php#L88-L90